crates/ruff_linter/src/codes.rs

```diff
@@ -963,6 +963,7 @@ pub fn code_to_rule(linter: Linter, code: &str) -> Option<(RuleGroup, Rule)> {
         (Ruff, "033") => (RuleGroup::Preview, rules::ruff::rules::PostInitDefault),
         (Ruff, "100") => (RuleGroup::Stable, rules::ruff::rules::UnusedNOQA),
         (Ruff, "101") => (RuleGroup::Stable, rules::ruff::rules::RedirectedNOQA),
+        (Ruff, "102") => (RuleGroup::Preview, rules::ruff::rules::RedundantTernary),
```

Comment:
The `1XX` ruff rules are reserved for noqa specific rules. Let's change the code to 034.

Reply:
@RubenVanEldik I think this answers it 😄. This is also my first time contributing here.